src/modules/highlight/common/events.ts

```diff
@@ -19,152 +19,25 @@
 
 import { IDocInfo } from "./document";
 import { IHighlight, IHighlightDefinition } from "./highlight";
-// import { IPaginationInfo } from "./pagination";
 import { ISelectionInfo } from "./selection";
 import { Locator } from "../../../model/Locator";
 
-// in RENDERER: webview.send()
-// in WEBVIEW: ipcRenderer.on()
-export const R2_EVENT_SCROLLTO = "R2_EVENT_SCROLLTO";
-// tslint:disable-next-line:class-name
-export interface IEventPayload_R2_EVENT_SCROLLTO {
-  goto: string | undefined;
-  hash: string | undefined;
-  previous: boolean;
-}
-
-// in RENDERER: webview.send()
-// in WEBVIEW: ipcRenderer.on()
-export const R2_EVENT_PAGE_TURN = "R2_EVENT_PAGE_TURN";
-
-// in WEBVIEW: ipcRenderer.sendToHost()
-// in RENDERER: webview.addEventListener("ipc-message")
-export const R2_EVENT_PAGE_TURN_RES = "R2_EVENT_PAGE_TURN_RES";
-// tslint:disable-next-line:class-name
-export interface IEventPayload_R2_EVENT_PAGE_TURN {
-  direction: string;
-  go: string;
-}
-
-// in WEBVIEW: ipcRenderer.sendToHost()
-// in RENDERER: webview.addEventListener("ipc-message")
-export const R2_EVENT_READING_LOCATION = "R2_EVENT_READING_LOCATION";
-
-// tslint:disable-next-line:class-name
```

Comment:
This looks like a pretty big change. Not sure if it was intended or not, but just pointing it out in case it wasn't supposed to be here. If it was unused then feel free to ignore me : )

Reply:
none of this was actually used